inst/gran_source_list.tsv

```diff
@@ -34,6 +34,6 @@ usgs-r/grithub	0.10.0
 usgs-vizlab/vizlab	v0.1.7
 usgs-r/USGSHydroTools	v1.2.1
 usgs-r/loadflex	v1.1.11
-usgs-r/WREG	v2.01
+usgs-r/WREG	v2.02
```

Comment:
Looks like the release tag on github is "2.02", not "v2.02". You can just delete the "v" in this file and push (you don't need to close this pr and submit another).